draft-ietf-quic-transport.md

```diff
@@ -1894,8 +1894,14 @@ use the server to send more data toward the victim than it would be able to send
 on its own.
 
 The primary defense against amplification attack is verifying that an endpoint
```

Comment:
This should be "the peer" instead of "an endpoint", since the context here is that this endpoint is doing the validation.